.github/workflows/ci.yml

```diff
@@ -32,3 +32,6 @@ jobs:
 
       - name: Run tests
         run: spago test --no-install
+
+      - name: Check docs
+        uses: colinwahl/purescript-doc-check-action@main
```

Comment:
Do you anticipate any breaking changes to the tool / new required fields in the action.yml coming up that might break repositories downstream? If you're still iterating on this I'd just like to make sure CI doesn't break going forward.